lib/forever/cli.js

```diff
@@ -554,7 +554,6 @@ app.cmd('columns add :name', cli.addColumn = function (name) {
     columns.push(name);
 
     forever.config.set('columns', columns);
-    forever.config.saveSync();
```

Comment:
Why are all of save calls removed without any replacement?

Reply:
Configstore persists the added configs right away, it doesn't require explicit save method call to save it to the file.